--- c:\Users\tinn\Documents\SMARTUR\DEVELOPMENT\MOBILE\lib\presentation\screens\auth\welcome_screen.dart
+++ c:\Users\tinn\Documents\SMARTUR\DEVELOPMENT\MOBILE\lib\presentation\screens\auth\welcome_screen.dart
@@ -584,8 +584,7 @@
                                 ),
                         ),
                       ],
-           ],
-                      if (!isWaitingOTP) ...[
+                      if (!isWaitingOTP && !isForgotPassword) ...[
                         const SizedBox(height: 12),
                         OutlinedButton(
